src/sql/azdata.d.ts

```diff
@@ -4927,6 +4927,7 @@ declare module 'azdata' {
 
 		export interface ICellOutputMetadata {
 			azdata_chartOptions?: any;
+			displayActionBar?: boolean;
```

Comment:
this should be added to the azdata.proposed.d.ts first

Reply:
Moved the action bar changes its own PR: https://github.com/microsoft/azuredatastudio/pull/16178